experimental/torch_xla2/torch_xla2/ops/jaten.py

```diff
@@ -357,6 +357,12 @@ def repeat_interleave(repeats, dim=0):
   return jnp.repeat(jnp.arange(repeats.shape[dim]), repeats)
 
 
+# aten.upsample_bilinear2d
+@op(torch.ops.aten.upsample_bilinear2d)
```

Comment:
Can you also enable the tests in `test_ops.py` for this op?

Reply:
thanks, this op don't exist in `test/test_ops.py` now, do we want to add external test for this op?